iml-wasm-components/iml-input/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+#[macro_use]
```

Comment:
Shouldn't need this: https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html